spec/models/option_set_spec.rb

```diff
@@ -0,0 +1,42 @@
+require 'rails_helper'
+
+describe OptionSet, type: :model do
+
+  before do
+    seed_data
+    #UNUSED AdHocCacheManager.clear_all
+  end
+
+  it 'has a valid factory' do
+    expect(create(:option_set)).to be_valid
+  end
+
+  it 'can fetch instance from including class' do
+    expect(OptionSet.fetch(Loan, :status)).to be
+  end
+
+  context 'seed data' do
```

Comment:
I don't think we should be testing the seed data. We don't need to. Seed data is for setting up a new environment. A failure in seeds shouldn't cause problems in production.